tests/tests_signin_user.py

```diff
@@ -0,0 +1,13 @@
+from django.test import Client
+
+
+def test_with_authenticated_client(client, django_user_model):
+	username = "test"
+	password = "test"
+	user = django_user_model.objects.create_user(username=username, password=password)
+	client.force_login(user)
+	respons = client.get('/personal_area', follow=True)
+	assert respons.status_code == 200
+	c = Client()
+	respons2 = c.post('/sign_in/', {'username': 'john', 'password': 'smith'})
```

Comment:
I think you missed it :)
john smith, is not test test user

Reply:
This is the implementation of Client, no need to run sign in,
you have it already, this is client 
You should replace all the line below, with single assert
assert respons.content.find(b"run your job search now") != -1

I sent the example how to check the option received in clinet response, FYI